coreservices/registration/src/CatenaX.NetworkServices.Registration.Service/Startup.cs

```diff
@@ -48,6 +50,14 @@ public void ConfigureServices(IServiceCollection services)
             
             services.AddTransient<IRegistrationBusinessLogic, RegistrationBusinessLogic>();
             services.AddTransient<IRegistrationDBAccess, RegistrationDBAccess>();
+
+            services.AddCustodianService(Configuration.GetSection("Custodian"));
+
+            services.AddTransient<IBPNAccess, BPNAccess>();
+            services.AddHttpClient("bpn", c =>
```

Comment:
creation of service-specific dependencies should be encapsulated like in class CustodianServiceCollectionExtension